Amend salaried teacher training non-UK content (#4181)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

      <p>
      Learn more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.
      </p>
  salaried-teacher-training:
    title: salaried teacher training
    text: |-
      <p>

>       You can apply for a salaried teacher training course. However, these courses are limited in number and very competitive.

Reviewer: Just seeing if we need 'in number', we don't have it in the sentence below, just thinking we don't need it here either?

Author: ```suggestion
      You can apply for a salaried teacher training course. However, places on these courses are limited and very competitive.
```
